mysql-test/t/percona_bug_ps1095.test

```diff
@@ -0,0 +1,16 @@
+# Bug PS-1095: Issues then thread_pool_stall_timeout is above 2.1 seconds
+
+-- source include/have_pool_of_threads.inc
+
+CREATE TABLE t1 (i int);
+
+--sleep 9
+
+# The server should be disconnected by the timer thread
```

Comment:
Don't assume that in our Jenkins the timer thread will be scheduled to be run at least once, but synchronize: replace --error/drop table with wait_until_disconnected.inc